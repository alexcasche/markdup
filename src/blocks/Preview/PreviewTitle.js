import styled from 'styled-components'
import { shade } from 'polished'

const PreviewTitle = styled.p`
  margin: 0;
  color: ${props => shade(0.9, props.theme.colors.grey)};
  font-size: 12px;
  font-weight: 500;
  text-transform: uppercase;
  flex-grow: 1;
`

export default PreviewTitle
