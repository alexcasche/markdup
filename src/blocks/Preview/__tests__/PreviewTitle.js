import PreviewTitle from '../PreviewTitle'

describe('PreviewTitle Element', () => {
  it('renders correctly', () => {
    const tree = render(<PreviewTitle>Preview</PreviewTitle>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
