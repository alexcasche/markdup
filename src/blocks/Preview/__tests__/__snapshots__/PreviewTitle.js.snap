// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreviewTitle Element renders correctly 1`] = `
.c0 {
  margin: 0;
  color: #999ba1;
  font-size: 12px;
  font-weight: 500;
  text-transform: uppercase;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

<p
  className="c0"
>
  Preview
</p>
`;
